src/styles/main.css

```
@@ -497,6 +497,38 @@ html, body {
+.passkeys-empty, .passkeys-error {
+  font-size: 13px; color: var(--muted); padding: 4px 0 8px;
+}
+.passkeys-error { color: var(--danger); }
+.passkeys-row {
+  display: flex; align-items: center; justify-content: space-between; gap: 10px;
+  padding: 10px 12px; margin-bottom: 6px; background: var(--card-bg);
+  border-radius: var(--radius-sm); border: 1px solid var(--border);
+}
+.passkeys-row-text { min-width: 0; flex: 1; }
+.passkeys-row-title { font-size: 14px; font-weight: 500; }
+.passkeys-row-sub  { font-size: 11px; color: var(--muted); margin-top: 2px; word-break: break-word; }
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)